api/v1/api.go

```diff
@@ -59,6 +59,21 @@ func (a *Api) Register(container *restful.Container) {
 		Operation("exportmetricsSchema").
 		Writes(TimeseriesSchema{}))
 	container.Add(ws)
+	ws = new(restful.WebService)
+	ws.Path("/api/v1/sinks").
+		Doc("Sinks currently in use").
+		Produces(restful.MIME_JSON)
+	ws.Route(ws.POST("").
+		To(a.setSinks).
+		Doc("set the current sinks").
```

Comment:
I guess we don't have to add anything here, we're as verbose as we are elsewhere. We should have more docs for it though. Questions always come :)